packages/synapse-react-client/src/components/row_renderers/Dataset_V2.tsx

```diff
@@ -0,0 +1,132 @@
+import { Component, MouseEvent } from 'react'
+import { DATASET } from '../../utils/SynapseConstants'
+import { calculateFriendlyFileSize } from '../../utils/functions/calculateFriendlyFileSize'
+import * as Utils from './utils'
+import { GenericCardSchema, getValueOrMultiValue } from '../GenericCard'
+import { SelectColumn, ColumnModel } from '@sage-bionetworks/synapse-types'
+import { PRODUCTION_ENDPOINT_CONFIG } from '../../utils/functions/getEndpoint'
+import { QueryVisualizationContextType } from '../QueryVisualizationWrapper'
+import { AddToDownloadCart } from '../AddToDownloadCart'
+import { SYNAPSE_ENTITY_ID_REGEX } from '../../utils/functions/RegularExpressions'
+
+export type Dataset_V2_Props = {
+  data?: any
+  schema?: any
+  secondaryLabelLimit?: number
+  selectColumns?: SelectColumn[]
+  columnModels?: ColumnModel[]
+  genericCardSchema?: GenericCardSchema
+  queryVisualizationContext: QueryVisualizationContextType
+}
+
+class Dataset_V2 extends Component<Dataset_V2_Props, never> {
```

Comment:
Why a class component? Please convert to a function component--it should be pretty straightforward since this component is stateless.

Reply:
Also nit but remove underscores for consistency
```suggestion
class DatasetV2 extends Component<DatasetV2Props, never> {
```